exercises/webapp/src/components/HackerNews.spec.js

```diff
@@ -0,0 +1,70 @@
+import {
+  mount
+} from '@vue/test-utils';
+import HackerNews from './HackerNews.vue';
+import NewsItem from './NewsItem.vue';
+
+describe("Sorting Order", () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(HackerNews, {
+      propsData: {
+        initialNews: [{
+          id: 0,
+          title: "Message 1",
+          voteCount: 2
+        }, {
+          id: 1,
+          title: "Message 2",
+          voteCount: 4
+        }],
+      }
+    });
+  });
+
+  it("sorting order", async () => {
+    expect(wrapper.findAllComponents(NewsItem).at(0).text()).toContain("Message 2");
+    expect(wrapper.findAllComponents(NewsItem).at(1).text()).toContain("Message 1");
+
+    await wrapper.find("#switch-order-button").trigger("click");
+
+    expect(wrapper.findAllComponents(NewsItem).at(0).text()).toContain("Message 1");
+    expect(wrapper.findAllComponents(NewsItem).at(1).text()).toContain("Message 2");
+
+    await wrapper.find("#switch-order-button").trigger("click");
+
+    expect(wrapper.findAllComponents(NewsItem).at(0).text()).toContain("Message 2");
+    expect(wrapper.findAllComponents(NewsItem).at(1).text()).toContain("Message 1");
```

Comment:
This is fine, however, some other groups had the idea of assigning a CSS class to the list item component. So they could write less verbose tests like:
```js
    expect(wrapper.findAll('h2.news-message').wrappers.map(w => w.text())).toEqual(['Message 2', 'Message 1');
    await wrapper.find("#switch-order-button").trigger("click");
    expect(wrapper.findAll('h2.news-message').wrappers.map(w => w.text())).toEqual(['Message 1', 'Message 2'');